examples/microsoft_devicecode_tenant_user.rs

```diff
@@ -0,0 +1,44 @@
+use oauth2::basic::BasicClient;
+use oauth2::devicecode::StandardDeviceAuthorizationResponse;
+use oauth2::reqwest::async_http_client;
+use oauth2::{AuthUrl, ClientId, DeviceAuthorizationUrl, Scope, TokenUrl};
+use std::error::Error;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let device_auth_url = DeviceAuthorizationUrl::new(
+        "https://login.microsoftonline.com/{tenant}/oauth2/v2.0/devicecode".to_string(),
```

Comment:
to make it clear to users that this needs to be substituted for their tenant, I'd suggest defining a `TENANT_ID` global at the top, and then using the `format!` macro here to automatically fill it in so that they only have to update it in one place at the top. and then also a comment next to that global instructing them to do so, maybe with a link for where they can find that tenant ID

Reply:
Done